types/supabase.ts

```
@@ -764,10 +770,10 @@ export type Database = {
         Args: {
           p_student_id: string;
         };
-        Returns: Array<{
+        Returns: {
           area_requirements_fulfilled: boolean;
           elective_courses_fulfilled: boolean;
-        }>;
+        }[];
```

ESLint

[error] Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.

(@typescript-eslint/array-type)